src/vulkan-renderer/world/cube.cpp

```diff
@@ -1,7 +1,9 @@
 #include <inexor/vulkan-renderer/world/cube.hpp>
+#include <utility>
 
 namespace inexor::vulkan_renderer::world {
 void Indentation::set(std::optional<std::uint8_t> x, std::optional<std::uint8_t> y, std::optional<std::uint8_t> z) {
+    assert(x <= MAX_INDENTATION && y <= MAX_INDENTATION && z <= MAX_INDENTATION);
     if (x != std::nullopt) {
```

Comment:
`std::optional` overrides the bool operator so you can remove the `!= std::nullopt` bit.

Reply:
Another way would be `x.has_value()`.